chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
This function has some issues. 

1. A user can have multiple ROLEs, but if they do, the call to create a surveillance will only ever pass in one ROLE, as chosen by the user doing the activity. Assuming that ROLE is one of the correct ones to use, the system should use that passed in role. If it's not one of the Authorized roles the system should return an "Unauthorized" error.
2. CMS_STAFF has no ability to do anything about Surveillances. There are three ROLEs that are allowed to do stuff with Surveillance. ROLE_ADMIN, ROLE_ACB_ADMIN, and ROLE_ACB_STAFF. Of those three, ROLE_ADMIN is allowed to do everything, but the other two are the ACB versions, so they are not allowed to edit/delete a surveillance that was created by ROLE_ADMIN
3. ROLE_ONC_STAFF should also not be here at all. That ROLE is only for viewing reports & downloading the CMS file. It should have no ability to edit or modify any listings